core-services/prow/02_config/opendatahub-io/opendatahub-operator/_prowconfig.yaml

```diff
@@ -13,6 +13,7 @@ tide:
     - do-not-merge/hold
     - do-not-merge/invalid-owners-file
     - do-not-merge/work-in-progress
+    - jira/invalid-bug
```

Comment:
This needed? We don't really link our PRs to Jiras...

Reply:
saw it in workbenches's [PR](https://github.com/openshift/release/pull/43275/files), Ican remove it if not in use

Reply:
Ah -- I didn't vet theirs super closely. I think you just need the transfer issue functionality for us to use the bot 🤔

Reply:
Um -- come to think of it -- I think you're requiring another label on _all_ PRs. Is that intended?